### PR TITLE
Re-opened pull request for sound in whiteboard

### LINKED ISSUE #36: Add song from Youtube to play on whiteboard
Enable players to choose a song to upload from YouTube. Once they uploaded a song,they must then click a button to play it.After this they have the option to pause, continue playing, and to upload a new song from Youtube. To do this you will be using YouTube APIs and following Nick's tutorial.

### PR DESCRIPTION
**Pull request by navila-luna**

### CODE REVIEW

**Review by marcoanaya — COMMENTED** [review]
Looks good and functional to me. Seems like its integrated into the tool bar Hailey made, and with a similar approach/style

File: public/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <!-- <link rel="icon" type="image/png" href="images/favicon.png"> -->
     <link rel="stylesheet" href="css/styles.css">
     <link rel="stylesheet" href="css/draggable.css">
-
+    <script src="../js/libs/p5.min.js"></script>
```
Comment (nbriz) [inline public/index.html]: did u mean to add this here? don't think it's necessary.

File: public/whiteboard/index.html [hunk 2]
```diff
@@ -23,12 +23,14 @@
     <script src="../js/libs/p5.min.js"></script>
     <script src="../js/utils/draggable.js"></script>
     <script src="/socket.io/socket.io.js"></script>
+    <script src="https://www.youtube.com/iframe_api"></script>
+    <script src="js/YTSampler.js" ></script>
```
Comment (nbriz) [inline public/whiteboard/index.html]: this is looking for a file called `YTSampler.js` (capital "S"), but ur file is actually called `YTsampler.js` (lower case "s"), so the js file isn't actually getting loaded up for me. i was able to change the name locally to test if it works (&& it does! solid job!) but i wonder how u got it to work on ur end w/the mis-matched file names? maybe u've got a local commit u haven't pushed yet?

**Review by nbriz — CHANGES_REQUESTED** [review]
everything looks good to me, except for the couple of comments i made below (the second one being most important), after u address those feel free to merge!

**Pull request merged**